app/views/admin/dashboard/imports/index.haml

```diff
@@ -1,3 +1,19 @@
+-if can_edit_users? || can_edit_translations? || can_administer_health?
```

Comment:
@everestwitman can you refactor these tabs into a single partial and include it where appropriate?  You may want to use https://apidock.com/rails/ActionView/Helpers/UrlHelper/current_page%3F  to determine which should be active, it's probably cleaner than controller path.